internal/compiler/typeregister.rs

```diff
@@ -396,3 +400,15 @@ impl TypeRegister {
         }
     }
 }
+
+pub fn logical_point_type() -> Type {
```

Comment:
There is already a `Point` structure declared in the builtins.slint.   Does that re-use it or create another Point?

Reply:
This creates another Point type which is identical, because we can't access the type registry / loaded builtin types from within `fn BuiltinFunction::ty(&self) -> Type` :(

Reply:
Yeah, that's unfortunate.  Maybe we could do something like we do for the enums or some other kind of getter. But that's not worth it for this patch.

This still deserve a comment though, i'd say.